pwiz_tools/Skyline/SettingsUI/ViewLibraryPepInfoList.cs

```diff
@@ -278,8 +313,8 @@ public IList<int> Filter(string filterText, string filterCategory)
             // We have to deal with the UnmodifiedTargetText separately from the adduct because the
             // adduct has special sorting which is different than the way adduct.ToString() would sort.
 
-            // Find the indices of entries that have a field that could match the search term if something was appended to it 
-            var filteredIndices = PrefixSearchByProperty(filterText);
+            // Find the indices of entries that have a field that contain the search term
+            var filteredIndices = ContainsSearchByProperty(filterText);
```

Comment:
This would be a performance hit for the conventional use case. But changing PrefixSearchByProperty to return and empty list when filterText starts with "*" would be useful.